### ISSUE TITLE
🐛 Bug Report: getSessions response (sessions attribute) is not an array after deleting one session anymore.

### ISSUE THREAD

**Issue by XeroxDev** [appwrite/appwrite#3083]
### 👟 Reproduction steps

1. Create multiple sessions
2. Call `/v1/account/sessions` and see response
3. Delete an "older" session
4. Call `/v1/account/sessions` again and see the change

This is getting fixed, after a new session is created.

### 👍 Expected behavior

`sessions` should stay an array.
```json
{
    "total": 2,
    "sessions": [
        {
            ...
        },
        {
            ...
        }
    ]
}
```

### 👎 Actual Behavior

`sessions` is getting converted to an object.

```json
{
    "total": 2,
    "sessions": {
        "0": {
            ...
        },
        "2": {
            ...
        }
    }
}
```

### 🎲 Appwrite version

Version 0.13.x

### 💻 Operating system

Linux

### 👀 Have you spent some time to check if this issue has been raised before?

- [X] I checked and didn't find similar issue

### 🏢 Have you read the Code of Conduct?

- [X] I have read the [Code of Conduct](https://github.com/appwrite/appwrite/blob/HEAD/CODE_OF_CONDUCT.md)

**Comment by stnguyen90** [comment]
Looks like [`unset($sessions[$key])`](https://github.com/appwrite/appwrite/blob/master/app/controllers/api/account.php#L1610) can convert `$sessions` into an associative array leading to:

```json
{
    "total": 2,
    "sessions": {
        "0": {
            ...
        },
        "2": {
            ...
        }
    }
}
```

It might be best to call `\array_values()` before inserting back into the DB as suggested by [Stack Overflow](https://stackoverflow.com/questions/369602/deleting-an-element-from-an-array-in-php).

**Comment by christyjacob4** [comment]
@XeroxDev  thanks for raising this, we will take a look 👍

**Comment by eldadfux** [comment]
> @XeroxDev thanks for raising this, we will take a look 👍

It should get fixed once @Meldiron completes the PR to get sessions using a DB filter.

**Comment by TorstenDittmann** [comment]
This got implemented and released with Appwrite 1.0.0 🥳 

Thank you for your contribution 🙏🏻  You can read more [here](https://appwrite.io/1.0).
